src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/CQLSAIIndex.java

```diff
@@ -128,29 +135,68 @@ static IndexTarget indexTarget(IndexMetadata indexMetadata)
       throws UnknownCqlIndexFunctionException, UnsupportedCqlIndexException {
     Objects.requireNonNull(indexMetadata, "indexMetadata must not be null");
 
-    var target = CQLSAIIndex.Options.TARGET.getWithDefault(indexMetadata.getOptions());
-    Matcher matcher = INDEX_TARGET_PATTERN.matcher(target);
-    if (!matcher.matches()) {
-      throw new UnsupportedCqlIndexException(
-          "Could not parse index target: '" + target + "'", indexMetadata);
-    }
-
+    // if the regex matches then the target is in the form "keys(foo)", "entries(bar)" etc
+    // if not, then it must be a simple column name and implicitly its type is VALUES
+    var target = indexMetadata.getTarget();
+    Matcher matcher = TARGET_REGEX.matcher(target);
     String columnName;
-    String functionName;
-    if (matcher.group(2) == null) {
-      columnName = matcher.group(1);
-      functionName = null;
-    } else {
+    CqlIndexType cqlIndexType;
+    if (matcher.matches()) {
+      cqlIndexType = CqlIndexType.fromString(matcher.group(1));
       columnName = matcher.group(2);
-      functionName = matcher.group(1);
+    } else {
+      columnName = target;
+      cqlIndexType = CqlIndexType.VALUES;
```

Comment:
I am not sure about this, why is the default Values ? 
What about what this is an index on a text field ?